core/unit_test/Makefile

```diff
@@ -306,6 +306,8 @@ endif
 
 ifeq ($(KOKKOS_INTERNAL_USE_SERIAL), 1)
     OBJ_SERIAL = UnitTestMainInit.o gtest-all.o
+    OBJ_SERIAL += TestSerial_Incremental_01_ExecutionSpace.o
```

Comment:
I don't think that incremental tests need to be added to the existing targets in the GNU Makefile